numba/tests/test_errorhandling.py

```diff
@@ -465,5 +465,39 @@ def foo(x):
             self.assertIn(expected, str(raises.exception))
 
 
+class TestCurlyBracesInPaths(unittest.TestCase):
+    """
+    Test that error messages handle file paths with curly braces (issue #10094)
+    """
+
+    def test_placeholders_with_positional_args(self):
+        from numba.core.errors import _format_msg
```

Comment:
Can the import go to the to of the file? I don't see a good reason to delay the import, but maybe I missed something?